client/my-sites/domains/domain-management/transfer/transfer-out/locked.jsx

```diff
@@ -47,23 +46,25 @@ class Locked extends React.Component {
 	}
 
 	render() {
-		const { translate } = this.props;
-		const { privateDomain } = getSelectedDomain( this.props );
+		const { translate, selectedSite } = this.props;
+		const { domain: domainName, privateDomain, registrar } = getSelectedDomain( this.props );
 
 		return (
 			<div>
 				<Card className="transfer-out__card">
 					<p>
-						{ privateDomain
+						{ privateDomain && registrar === registrarNames.WWD
 							? translate(
-									'To transfer your domain, we must unlock it and remove Privacy Protection. ' +
-										'Your contact information will be publicly available during the transfer period.'
+									'To transfer your domain, we must unlock it and remove Privacy Protection. Your contact information will be publicly available during the transfer period. The domain will remain unlocked and your contact information will be publicly available until the transfer is canceled or completed.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'To transfer your domain, we must unlock it and remove Privacy Protection. Your contact information will be publicly available during the transfer period.' )` *ES Score: 8*